mesheryctl/internal/cli/root/system/start.go

```diff
@@ -105,13 +105,18 @@ func start() error {
 		return err
 	}
 
-	if utils.PlatformFlag == "docker" || utils.PlatformFlag == "kubernetes" {
-		currCtx.SetPlatform(utils.PlatformFlag)
-	} else {
-		if utils.CfgFile != utils.DefaultConfigPath {
+	if utils.PlatformFlag == "" {
+		utils.PlatformFlag = currCtx.GetPlatform()
+	}
```

Comment:
@nehaLohia27  This part is unnecessary, as we have already maintained this thing later ahead.

Reply:
@piyushsingariya I removed this code and pushed changes

Reply:
@piyushsingariya one lint error is coming.   
Warning: var-declaration: should omit type string from declaration of var tempContext; it will be inferred from the right-hand side (revive)

this is the code snippet. 
	if tempContext != "" {
		err = mctlCfg.SetCurrentContext(tempContext)
		if err != nil {
			return errors.Wrap(err, "failed to set temporary context")
		}
	}
any idea?

Reply:
`tempContext` we do not need to explicitly define the data type of this variable as it will be figured out from its initial assignment.